src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -164,6 +168,53 @@ const HomeViewArticlesSectionType = new GraphQLObjectType<any, ResolverContext>(
   }
 )
 
+export const ExploreByMarketingCategoriesType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "ExploreByMarketingCategories",
+  description: "A list of categories to explore",
```

Comment:
**thought (non-blocking):** Should we extract this new MP type into a separate location, rather than leaving it inline here? 

(Feel free to ignore this for now — I think this is another sign we need to re-org this file and break it up into separate files per section-type. Then this new type could continue to live alongside `HomeViewMarketingCollectionCategoriesSectionType`)

Reply:
I paired with @MounirDhahri this week and he said to leave it here since we don't have a better place to have it right now.
I was trying to let it live under `section/ExploreByCategories.ts` but we were getting undefined due to the type not being ready on time, I think I will just ignore and wait for more organisations that are happening, we def need to come back to this section in at least 2 PRs, so I'll make sure to re-check it 👍

Reply:
Thanks, it's fine to wait for the re-org